### DISCUSSION TITLE
Flashing File with STM

### DISCUSSION THREAD

**Discussion by dedvoltg** [erichelgeson/BlueSCSI-v1#174] [Q&A]
I'm just wondering because ill be making my own blue scsi and I will be flashing my own blue pill from scratch and I want to know the correct file to use now for this and for future use.

**Comment by erichelgeson** [comment]
The wiki article on Flashing will bring you to the BlueSCSI Updater https://github.com/erichelgeson/BlueSCSI-Updater - which will be what you need to flash the USB bootloader and then load the firmware with it. HTH and goodluck on building your own!

**Reply by dedvoltg** [reply]
Thank you for your reply!
I just want to confirm after flashing it with option 2 on macOS now it is complete correct or is there something else I need to do to it?

**Reply by erichelgeson** [reply]
Selecting Option 2 in the Updater app will flash the USB bootloader, then flash the firmware, you should be good to go - the easiest way to check is just stick an SD card in there then power it up, look at the LOG.txt to validate it's good to go.

**Reply by dedvoltg** [reply]
Ok thank you for all the help on this! 
Can't wait to use it!
